internal/cri/server/container_stats_list.go

```diff
@@ -538,3 +549,68 @@ func (c *criService) memoryContainerStats(ID string, stats interface{}, timestam
 	}
 	return nil, nil
 }
+
+func (c *criService) memoryContainerSwap(ID string, stats interface{}, timestamp time.Time) (*runtime.SwapUsage, error) {
+	switch metrics := stats.(type) {
+	case *cg1.Metrics:
+		if metrics.Memory != nil && metrics.Memory.Swap != nil {
+			swap := metrics.Memory.Swap
```

Comment:
Can we use the `GetMemory()`, `GetSwap()` methods etc which are defined in the api?